src/selectors_vm/stack.rs

```diff
@@ -3,7 +3,7 @@ use super::program::AddressRange;
 use super::ast::NthChild;
 use crate::html::{LocalName, Namespace, Tag};
 use crate::memory::{LimitedVec, MemoryLimitExceededError, SharedMemoryLimiter};
-use hashbrown::{HashSet, HashMap};
+use hashbrown::{HashSet, HashMap, hash_map::RawEntryMut};
```

Comment:
Rust uses hashbrown internally since 1.36, so we can use std HashMap: https://github.com/rust-lang/rust/blob/master/RELEASES.md#libraries-8

Reply:
Raw entry APIs are unstable, so we have to explicitly use hashbrown if we want to use its raw entry API.

Reply:
Let's leave a comment, so we'll switch to std once it stabilise

Reply:
Though... I might be missing something, but why we can't just use [HashMap::entry()](https://doc.rust-lang.org/beta/std/collections/struct.HashMap.html#method.entry) here?

Reply:
The lifetimes don't check out, so we have to make an owned version of the local name to access the entry even if we don't actually need an owned key to insert.

Reply:
I see, let's just add the comment then.